_posts/2020-06-28-openfaas-loki-into.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Time travelling through your OpenFaaS Function logs
+description: "In this post Lucas will introduce the openfaas-loki log provider and how to use it to get historic logs for your OpenFaaS functions."
+date: 2020-05-28
+image: /images/2020-openfaas-loki/2020-06-loki-logs-timey-wobbly.png
+categories:
+  - logs
+  - kubernetes
+  - serverless
+  - tutorial
+  - grafana
+  - loki
+author_staff_member: lucas
+dark_background: true
+
+---
+
+In this post Lucas will introduce the [`openfaas-loki`](https://github.com/LucasRoesler/openfaas-loki) [log provider](https://docs.openfaas.com/architecture/logs-provider/) and how to use it to get historic logs for your OpenFaaS functions.
+
+In any production service, logs are a critical component of observability and debuggability of the system. OpenFaaS builds in support for logs in _every_ installation. But it can be even better by using a new log provider. In this post we will show you how to supercharge the logs by installing [Grafana's Loki](https://grafana.com/oss/loki) and configure OpenFaaS to use Loki.
```

Comment:
>In any production service, logs are a critical component of observability and debuggability of the system

We can use plain English here to make the sentence easier to read and say for international readers:

> Logs are critical to observing and debugging systems once in production.

Or:

> Logs are critical to observing and debugging production systems.